h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoModel.java

```diff
@@ -972,4 +977,15 @@ public SharedTreeGraph convert(final int treeNumber, final String treeClass) {
         return trees_result;
     }
 
+    private class SharedTreeModelDescriptor extends MojoModelDescriptor {
```

Comment:
What is this class good for?

Reply:
Provides variable importaces for SharedTreeMojoModel models. Basic MojoModel does not provide variable importances for all the models in general.

Reply:
Because .e.g. GLM does not have it. tree-based models are guaranteed to have it, and we aimed for GBM first, is it made sense to do it like this.

Reply:
Now, this class no longer exists. Commit `d7ace24` made it obsolete.